libs/types-rs/src/hmpb.rs

```diff
@@ -118,14 +157,65 @@ impl FromBitStreamWith<'_> for Metadata {
     }
 }
 
-const BALLOT_HASH_LENGTH: u32 = 20;
-const HEX_BYTES_PER_CHAR: u32 = 2;
+impl ToBitStreamWith<'_> for Metadata {
+    type Context = Election;
+    type Error = Error;
+
+    fn to_writer<W: bitstream_io::BitWrite + ?Sized>(
+        &self,
+        w: &mut W,
+        context: &Self::Context,
```

Comment:
Maybe s/context/election/ for clarity and also to match the BMD encoding function

Reply:
dab346e3